tests/phpunit/tests/theme/wpThemeJson.php

```diff
@@ -811,58 +834,13 @@ public function test_get_stylesheet_generates_proper_classes_and_css_vars_from_s
 		);
 	}
 
-	/**
-	 * @ticket 53175
-	 * @ticket 54336
-	 * @ticket 58550
-	 * @ticket 60936
-	 * @ticket 61165
-	 */
-	public function test_get_stylesheet_preset_values_are_marked_as_important() {
```

Comment:
This was essentially a duplicate of the other preset tests - if the CSS changes in any way, those tests will flag it so it's redundant to have this.